twilight-http/src/request/application/interaction/create_response.rs

```diff
@@ -61,6 +61,19 @@ impl IntoFuture for CreateResponse<'_> {
 
 impl TryIntoRequest for CreateResponse<'_> {
     fn try_into_request(self) -> Result<Request, Error> {
+        // Check if the title for a modal response is valid.
+        if let Some(modal_title) = self
+            .response
+            .data
+            .as_ref()
+            .and_then(|data| data.title.as_ref())
+        {
+            twilight_validate::component::modal_title_length(modal_title).map_err(|e| Error {
+                kind: ErrorType::BuildingRequest,
+                source: Some(Box::new(e)),
+            })?;
```

Comment:
I'm unsure if this is the way I should handle this, if not please let me know the desired solution.

Reply:
We don't do validation here at the moment... current intuition and prior art is that this should go in `CreateResponse::new` (which is a breaking change). But the downside there, too, is that it's a breaking change. Both of these are problems we're aiming to solve for 0.16, so I don't have a good answer here at the moment. cc @7596ff

Reply:
this request already accepts invalid data, so this isn't even a complete fix. my reflex is to just punt to 0.16 because this comes under the whole "validate at the end" paradigm. we can at least leave the changes in the validate crate to save this pr, but i don't want to do anything to the request at the moment